src/ui/view/dataverse/component/dataverseItemCard/dataverseItemCard.tsx

```diff
@@ -16,13 +15,16 @@ export type DataverseItemCardProps = {
   type: DataverseItem
   label: string
   description: string
+  tags?: Array<string>
```

Comment:
The store selector will return an empty array if no tags are fetched on the ontology
```suggestion
  tags: string[]
```